pkg/controller/migration/controller.go

```diff
@@ -89,6 +90,9 @@ func Add(mgr manager.Manager) error {
 		return err
 	}
 
+	// Gather migration metrics every 10 seconds
```

Comment:
The _every 10 seconds_ part of this comment can get out of sync with where the sleep actually happens.  Recommend this only be documented where it's controlled.